### PR TITLE
{bio}[foss/2019b] medaka v1.2.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by dagonzalezfo**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/medaka/medaka-1.2.0-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+builddependencies = [('Autotools', '20180311')]
+
+dependencies = [
+    ('Python', '3.7.4'),  # includes cffi, numpy
+    # TensorFlow 2.2.0 required by medaka, see requirements.txt
+    ('TensorFlow', '2.2.0', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('h5py', '2.10.0', versionsuffix),
+    # Pysam 0.16.0.1 is strictly required, see requirements.txt
+    ('Pysam', '0.16.0.1', '', ('GCC', '8.3.0')),
```
[the remark this answers is not in the document]
Comment (dagonzalezfo) [reply easybuild/easyconfigs/m/medaka/medaka-1.2.0-foss-2019b-Python-3.7.4.eb:26]: I have issues with the GitHub actions test with the Pysam dependency. Thus, as a first attempt to solve it, I define it in detail. However the problem persist. The solution was to extend easyconfig test exceptions for medaka to this version.  Therefore I should test it without this definition. I really appreciate your suggestion.

File: test/easyconfigs/easyconfigs.py [hunk 2]
```diff
@@ -282,12 +282,14 @@ def check_dep_vars(self, dep, dep_vars):
                 # medaka 1.1.* requires TensorFlow 2.2.0 (while other 2019b easyconfigs use TensorFlow 2.1.0 as dep);
```
Comment (migueldiascosta) [inline test/easyconfigs/easyconfigs.py:282]: can you also update the comment here to include medaka 1.2?

File: test/easyconfigs/easyconfigs.py [hunk 3]
```diff
@@ -282,12 +282,14 @@ def check_dep_vars(self, dep, dep_vars):
...
                 # TensorFlow 2.2.0 is also used as a dep for Horovod 0.19.5;
                 # decona 0.1.2 and NGSpeciesID 0.1.1.1 depend on medaka 1.1.3
-                ('2.2.0;', ['medaka-1.1.[13]-', 'Horovod-0.19.5-', 'decona-0.1.2-', 'NGSpeciesID-0.1.1.1-']),
+                ('2.2.0;', ['medaka-1.2.[0]-', 'medaka-1.1.[13]-', 'Horovod-0.19.5-', 'decona-0.1.2-',
+                            'NGSpeciesID-0.1.1.1-']),
             ],
             # medaka 1.1.* requires Pysam 0.16.0.1,
```
Comment (migueldiascosta) [inline test/easyconfigs/easyconfigs.py:288]: same as above

**Review by migueldiascosta — CHANGES_REQUESTED** [review]

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

File: test/easyconfigs/easyconfigs.py (hunk 2 above)
Comment (dagonzalezfo) [reply test/easyconfigs/easyconfigs.py]: Doen

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/1fb54e09eaa5740a26a7ae21f040bcae for a full test report.

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @dagonzalezfo!

**Pull request merged**